coil-svg/src/commonMain/kotlin/coil3/decode/SvgDecoder.kt

```diff
@@ -0,0 +1,11 @@
+package coil3.decode
+
+object SvgDecoder {
```

Comment:
Let's do an expect/actual class instead of factory functions:

```kotlin
expect class SvgDecoder @JvmOverloads constructor(
    source: ImageSource,
    options: Options,
    useViewBoundsAsIntrinsicSize: Boolean = true,
) : Decoder {
    class Factory : Decoder.Factory
}
```

Reply:
Done